wasm/prover/Cargo.toml

```diff
@@ -44,8 +44,8 @@ time = { version = "0.3.34", features = ["wasm-bindgen"] }
 # Used to calculate elapsed time.
 web-time = "1.0"
 
-tlsn-core = { git = "https://github.com/tlsnotary/tlsn.git", tag = "v0.1.0-alpha.4", package = "tlsn-core" }
-tlsn-prover = { git = "https://github.com/tlsnotary/tlsn.git", tag = "v0.1.0-alpha.4", package = "tlsn-prover", features = [
+tlsn-core = { git = "https://github.com/tlsnotary/tlsn.git", branch = "dev", package = "tlsn-core" }
```

Comment:
shouldn't this be `main`?

Reply:
@yuroitaki We will have to change this to `alpha.5`, but this allows us to already test that everything will work